apps/meteor/app/lib/server/functions/setUserActiveStatus.ts

```diff
@@ -101,7 +108,10 @@ export async function setUserActiveStatus(userId: string, active: boolean, confi
 	}
 
 	if (user.username) {
-		await Subscriptions.setArchivedByUsername(user.username, !active);
+		const setArchivedResponse = await Subscriptions.setArchivedByUsername(user.username, !active);
+		if (setArchivedResponse.modifiedCount) {
+			void notifyOnSubscriptionChangedByUserIdAndRoomType(user._id, 'd');
```

Comment:
this notify should be by name and not by `id`.. the reason is that we're deactivating `userA`, so we need to archive everyone else's subscriptions that had a DM with `userA`.. but this `notifyOnSubscriptionChangedByUserIdAndRoomType` is actually just notifying `userA` instead of everyone else.